src/Camera.cc

```diff
@@ -23,7 +24,7 @@ using namespace sdf;
 
 /// \brief String names for the pixel formats.
 /// \sa Image::PixelFormat.
-static std::array<std::string, 19> kPixelFormatNames =
+static const std::array<const char *, 19> kPixelFormatNames =
```

Comment:
Did `string_view` not work? I see it in `ParticleEmitter`, but not here. I'm okay with either, but it would be nice to make them all the same.

Reply:
The only difference is the `const`, I've just committed https://github.com/gazebosim/sdformat/pull/1264/commits/632570e4151919ac5e6f1b58d63088420d634757 to see how the macOs and win runners will react

Reply:
seems to be working!

Reply:
yeah, looks like the compiler only considers `const std::array<const std::string_view, ...>` as literal